README.md

```diff
@@ -102,6 +102,23 @@ cd ~/Documents/mpide/libraries/
 git clone https://github.com/sandeepmistry/arduino-BLEPeripheral BLEPeripheral
 ```
 
+### Flashing a SoftDevice
+
```

Comment:
I'm not too keen to duplicate stuff from the nRF5 read me here, what do you think of having a link to it?

Also, these steps are only needed when you also use the nRF5 core. Other scenarios include cores like the original RBL ones or when using the an nRF8001 shield with an Uno, Zero, etc.

Maybe we need an nRF5x core section?